firmware/thruna/BMS/Src/main.c

```diff
@@ -180,6 +180,12 @@ struct PrechargeRelay *  precharge_relay;
 struct TractiveSystem *  ts;
 struct Clock *           clock;
 struct Eeprom *          eeprom;
+// BMS checks DCM, PDM, FSM
+bool heartbeat_monitor_checklist[HEARTBEAT_BOARD_COUNT] = { [BMS_HEARTBEAT_BOARD] = false,
+                                                            [DCM_HEARTBEAT_BOARD] = true,
+                                                            [PDM_HEARTBEAT_BOARD] = true,
+                                                            [FSM_HEARTBEAT_BOARD] = true,
+                                                            [DIM_HEARTBEAT_BOARD] = false };
```

Comment:
Right now we're providing half of the config info in `main.c`, and the other half the in the state machine all states function. 
I don't like this because its messy, and also makes it easier to make a mistake where you provide mistmatched configs. Also also, now you need to define the setters/getters wherever you use the heartbeat driver (what if you wanted to call its functions from a different file?).
Let me know what you think, but I feel like refactoring it so you provide all configs here would be a nice change, and also simplify the state machine code a lot.

Reply:
All heartbeat can methods now live in the heartbeat monitor itself, and get set when the monitor is created. All the config info is now in main.c as requested! :D